### PR TITLE
Add treasurer position

### PR DESCRIPTION
**Pull request by synixebrett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ nameless
 PaC  
 Flashy Moves
 
+## Support Staff
+
```
Comment (nameless) [inline README.md]:
```suggestion
```

Comment (nameless) [reply README.md]: Add a space to all or remove spaces to this one to remain consistent

Comment (synixebrett) [reply README.md]: We need to add a space to all

Comment (nameless) [reply README.md]: Then add a space to all

Comment (synixebrett) [reply README.md]: Out of scope

Comment (nameless) [reply README.md]: ok then

**Pull request merged**